fpdf/line_break.py

```diff
@@ -185,28 +188,33 @@ def automatic_break(self, justify):
 
 
 class MultiLineBreak:
-    def __init__(self, styled_text_fragments, size_by_style, justify=False):
-
+    def __init__(
+        self, styled_text_fragments, size_by_style, justify=False, print_sh=False
+    ):
         self.styled_text_fragments = styled_text_fragments
-
         self.size_by_style = size_by_style
         self.justify = justify
-
+        self.print_sh = print_sh
         self.fragment_index = 0
         self.character_index = 0
 
     def _get_character_width(self, character, style=""):
-        if character == SOFT_HYPHEN:
+        if character == SOFT_HYPHEN and not self.print_sh:
             # HYPHEN is inserted instead of SOFT_HYPHEN
             character = HYPHEN
         return self.size_by_style(character, style)
 
-    def get_line_of_given_width(self, maximum_width):
+    # pylint: disable=too-many-return-statements
+    def get_line_of_given_width(self, maximum_width, no_wordsplit=False):
```

Comment:
Parameters with "no" in their names should be avoided, regarding code readability...
Could we replace this by `wordsplit=True` maybe?

Reply:
I named it this way because I like it when the default for an unusual condition is False... :wink:
But yeah, `wordsplit=True` is probably easier on the eyes.